smt/surrogate_models/qp.py

```diff
@@ -11,7 +11,7 @@
 import scipy
 from smt.surrogate_models.surrogate_model import SurrogateModel
 from smt.utils.caching import cached_operation
-
+from smt.utils.kriging_utils import standardization
```

Comment:
Well, `standardization` should be moved in a another module in utils? `utils/misc.py` ?